pipelines/_ingredients/cpBiasBootstrapLSST.yaml

```diff
@@ -0,0 +1,42 @@
+description: cp_pipe bootstrap bias calibration for LSST isr task
+tasks:
+  cpBiasBootstrapIsr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      connections.ccdExposure: "raw"
+      connections.outputExposure: "cpBiasBootstrapIsrExp"
+      expectWcs: false
+      doDefect: false
+      doBias: false
+      doDark: false
+      doFlat: false
+      doDiffNonLinearCorrection: false
+      doBootstrap: true
+      doDeferredCharge: false
+      doBrighterFatter: false
+      doLinearize: false
+      doCorrectGains: false
+      doApplyGains: false
+      doVariance: true
+      doSaturation: false
+      doSuspect: false
+      doCrosstalk: true
```

Comment:
The crosstalk calibs, if in electrons, include the gains used in the crosstalk.  But by default the quadratic term is term is turned off for bootstrap (see below).  So it will use the crosstalk gains if it needs to, otherwise it will use the PTC gains.  This was done in DM-46052, and tests were added there.